### PR TITLE
Update design and delete dataset on flamectl.

### PR DESCRIPTION
**Pull request by raresgaia123**
## Description

Please provide a meaningful description of what this change will do, or is for. Bonus points for including links to
related issues, other PRs, or technical references.

Note that by _not_ including a description, you are asking reviewers to do extra work to understand the context of this
change, which may lead to your PR taking much longer to review, or result in it not being reviewed at all.

## Type of Change

- [ ] Bug Fix
- [x] New Feature
- [ ] Breaking Change
- [ ] Refactor
- [ ] Documentation
- [ ] Other (please describe)

## Checklist

- [ ] I have read the [contributing guidelines](/CONTRIBUTING.md)
- [ ] Existing issues have been referenced (where applicable)
- [ ] I have verified this change is not present in other open pull requests
- [ ] Functionality is documented
- [ ] All code style checks pass
- [ ] New code contribution is covered by automated tests
- [ ] All new and existing tests pass

### CODE REVIEW

File: cmd/flamectl/resources/design/design.go [hunk 1]
```diff
@@ -164,3 +165,37 @@ func GetMany(params Params) error {
 
 	return nil
 }
+
+func Update(params Params) error {
+	// read design via conf file
+	jsonData, err := os.ReadFile(params.DesignFile)
```
Comment (myungjin) [inline cmd/flamectl/resources/design/design.go:171]: Let's discuss this approach separately.

File: cmd/flamectl/resources/design/design.go [hunk 2]
```diff
@@ -164,3 +165,37 @@ func GetMany(params Params) error {
...
+	}
+
+	// construct URL
+	uriMap := map[string]string{
+		constants.ParamUser:     params.User,
+		constants.ParamDesignID: params.DesignId,
+	}
+	url := restapi.CreateURL(params.Endpoint, restapi.UpdateDesignEndPoint, uriMap)
+
+	code, responseBody, err := restapi.HTTPPut(url, design, "application/json")
```
Comment (myungjin) [inline cmd/flamectl/resources/design/design.go:190]: Let's check the server behavior on this request. We would like to update entries with non-empty strings.

Comment (raresgaia123) [reply cmd/flamectl/resources/design/design.go:184]: I've created a new task for handling this.

File: cmd/flamectl/cmd/update_design.go [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/cisco-open/flame/cmd/flamectl/resources/design"
+)
+
+var updateDesignCmd = &cobra.Command{
+	Use:   "design <designId> --flags (--description --name)",
```
Comment (myungjin) [inline cmd/flamectl/cmd/update_design.go:26]: Please revise the use. Something like the following may look reasonable.
design <designId> [--description <desc>] [--name <name>]

**Review by myungjin — COMMENTED** [review]
left one comment.

**Review by myungjin — APPROVED** [review]
lgtm

**Pull request merged**